src/components/team-member-add/TeamMemberAdd.module.css

```diff
@@ -1,9 +1,17 @@
 .container {
+  display: flex;
+  flex-direction: column;
   margin-bottom: 25px;
 }
 
+@media only screen and (min-width: 901px) {
+  .container {
+    display: block;
+  }
+}
+
 .header {
-  order: 99;
+  order: 9;
```

Comment:
Do you really need the `order` property?

Reply:
It's used to place the elements on top as header on desktop, but I did find a more elegant solution now!